cmd/check-reaper/main.go

```diff
@@ -195,3 +220,58 @@ func deletePod(ctx context.Context, client *kubernetes.Clientset, pod v1.Pod) er
 	options := metav1.DeleteOptions{PropagationPolicy: &propagationForeground}
 	return client.CoreV1().Pods(pod.Namespace).Delete(ctx, pod.Name, options)
 }
+
+// podConditions returns true if conditions are met to be deleted for checker pod
+func podConditions(pod v1.Pod, duration float64, phase v1.PodPhase) bool {
```

Comment:
is this func being used anywhere?

Reply:
Does not look like it...I am not sure why this function is saying it is an addition though this is not code that I wrote...I moved it to near the bottom though and added func jobConditions right below it....but yes this can be deleted